models/Token.js

```diff
@@ -0,0 +1,45 @@
+const { Schema, model } = mongoose;
```

Comment:
instead should be like this 
`const { Schema, model } = require('mongoose');
`